src/main/java/com/foundation/SearchFilesMain.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * @(#)SearchFilesMain.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *  <p>
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ */
+
+package com.foundation;
+
+import com.foundation.controller.Controller;
+
+/**
+ * Class that calls MVC classes and run the application
+ *
+ * @author Marco Velasquez
+ * @version 1.0.
+ */
+public class SearchFilesMain {
+
+    /**
+     *  Main method which execute the application
+     *
+     * @param args parameters passed from command line
+     */
+    public static void main(String[] args) {
+
+        // View view = new View();                not implemented yet
+        // SearchModel model = new SearchModel(); not implemented yet
+        Controller control = new Controller(); /* Controller instance */
+        // controller.StartView();                not implemented yet
+        control.ValidateSearchCriteria();
```

Comment:
Why do we need to call this at this point? If it is not needed, please remove it